kconfig-hardened-check.py

```diff
@@ -34,8 +34,9 @@
 from collections import OrderedDict
 import re
 
+X86_32 = "x86_32"
 X86_64 = "x86_64"
-SUPPORTED_ARCHS = [ X86_64 ]
+SUPPORTED_ARCHS = [ X86_32, X86_64 ]
```

Comment:
I capitalized it since it is a "constant". That's what PEP 8 [suggests](https://www.python.org/dev/peps/pep-0008/#constants).

Renaming it to match the other globals also makes sense, though.